docs/lambda.asciidoc

```diff
@@ -1,150 +1,86 @@
-:framework: lambda
-
 [[lambda]]
 
 ifdef::env-github[]
 NOTE: For the best reading experience,
 please view this documentation at https://www.elastic.co/guide/en/apm/agent/nodejs/current/lambda.html[elastic.co]
 endif::[]
 
-=== Get started with Lambda
-
-experimental::[]
+=== Get started with AWS Lambda Node.js Functions
+:layer-section-type: with-agent
 
-Getting Elastic APM set up for your lambda functions is easy,
-and there are various ways you can tweak it to fit your needs.
-Follow the guide below to get started, and for more advanced topics,
-check out the <<api,API Reference>>.
+The Node.js APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
 
 [float]
-[[lambda-installation]]
-==== Installation
-
-Setting up your Lambda function is a two-step process.
+[[aws-lambda-nodejs-prerequisites]]
+==== Prerequisites
```

Comment:
In the other PRs, we've moved the Prerequisite section within the Quick Start heading.

Reply:
Done in https://github.com/elastic/apm-agent-nodejs/pull/2630/commits/94da3631aaf5d09a68b93024b24c29c9f2c72f04